src/test/ui/const-eval/union-const-eval-field.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0080]: constant evaluation error
+  --> $DIR/union-const-eval-field.rs:38:5
+   |
+LL |     const FIELD3: Field3 = unsafe { UNION.field3 }; //~ ERROR E0080
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ attempted to read undefined bytes
```

Comment:
Oh wow, this is taking a strong stance in one particular camp of unsafe code guidelines! Do we really want this to error so early?

Reply:
If you do `0u8 - 1` inside a constant you also get a lint about  an unusable constant. You can still publish your crate with these broken constants, but actually using them anywhere will produce erorrs.

Reply:
I have no idea what that statement has to do with my question... this is a hard error, is it not? It says ERROR in all-caps. How's this related to overflow lints?

Reply:
uh... indeed. I need to check that out... goot catch.

Reply:
I had a look at the code again. Using a broken constant to obtain a runtime value will error, not lint + produce an undef. So maybe we cannot do this PR because it would be a breaking change.